lalrpop/src/parser/lrgrammar.rs

```diff
@@ -14,6 +14,9 @@ extern crate lalrpop_util as ___lalrpop_util;
 use self::___lalrpop_util::state_machine as ___state_machine;
 extern crate core;
 extern crate alloc;
+mod generated_code {
+#![allow(non_snake_case, non_camel_case_types, unused_mut, unused_variables, unused_imports, unused_parens, dead_code, clippy::all)]
+use super::*;
 
 #[cfg_attr(rustfmt, rustfmt_skip)]
 mod ___parse___Top {
```

Comment:
I am not sure about this. Almost the same thing is attached on parse module, which is practically the entire contents of the `generated_code`. It seems a depth is added but almost nothing more covered.

Reply:
Yeah, I think you're right about that.  Trying without lines 18 and 19 seems to be working fine for me.  I'll push an update to remove those.

Reply:
Now the only contents of `generated_code` is `mod ___parse___Top` 😅

Reply:
Oh, I see your point.  Yeah, that does seem like an issue.

Of course, in a parser with more than one `pub` nonterminal, there would be multiple modules inside `generated_code`.  But even then, I guess the extra layer of `generated_code` may not add much value.

This seems to somewhat invalidate https://github.com/lalrpop/lalrpop/pull/739#issuecomment-1518482164.

Maybe a better solution would be to add `#[allow(clippy::all)]` directly on each parse mod?

Reply:
Yes, that makes sense.

Reply:
Okay, I'll abandon this PR and add a note in #769.  If I have time this weekend, I'll try the `#[allow(clippy::all)]` on the parse modules approach.